source/code/plugins/in_npmd_server.rb

```diff
@@ -79,12 +81,19 @@ def configure(conf)
         EXIT_RESTART_BACKOFF_TIMES_SECS = [60, 120, 300, 600, 1200, 2400]
         EXIT_RESTART_BACKOFF_THRES_SECS = 900
 
+        NPMD_AGENT_CAPABILITY = "cap_net_raw+ep"
+        NPMD_VERSION_FILE_DIR = "state"
+        NPMD_VERSION_FILE_NAME = "npm_version"
+
         def start
             @binary_presence_test_string = "npmd_agent" if @binary_presence_test_string.nil?
             @binary_invocation_cmd = @location_agent_binary if @binary_invocation_cmd.nil?
+            @npmd_version_dir_prefix = "/var/opt/microsoft/omsagent" if @npmd_version_dir_prefix.nil?
             kill_all_agent_instances()
             upload_pending_stderrors()
             check_and_update_binaries()
+            @do_capability_check = true unless @do_capability_check == false
+            check_agent_capability() unless !@do_capability_check
```

Comment:
For consistency, I would suggest using the same syntax to check `@do_capability_check`: either `@do_capability_check == false` or `!@do_capability_check`